aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunClientFactories.java

```diff
@@ -81,17 +88,82 @@ private static AliyunClientFactory loadClientFactory(
   }
 
   static class DefaultAliyunClientFactory implements AliyunClientFactory {
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultAliyunClientFactory.class);
```

Comment:
nit:  We ususally define the `LOG` in the class file rather than the inner class in apache iceberg.  Maybe we can move it to the `AliyunClientFactories`.